javascript/implementation/IndexedDbStore.ts

```diff
@@ -243,17 +252,23 @@ export class IndexedDbStore implements Store {
         return result;
     }
 
-    async claimChain(medallion: Medallion, chainStart: ChainStart, actorId?: ActorId): Promise<ClaimedChain> {
+    async getChainIdentity(chainInfo: [Medallion, ChainStart]): Promise<string> {
+        await this.ready;
+        const wrappedTransaction = this.getTransaction();
+        const identity = await wrappedTransaction.objectStore('identities').get(chainInfo);
+        return identity;
+    }
+
+    async claimChain(medallion: Medallion, chainStart: ChainStart, actorId?: ActorId, transaction?: Transaction): Promise<ClaimedChain> {
```

Comment:
The transaction goes out of scope and finalizes if the same transaction isn't passed along into claimChain.

Reply:
Yeah makes sense.  I feel like we should remove claimChain from the Store interface, because it'll only be used internally by addBundle (and this avoids us having the problem of needing to specify the type of transaction in the interface).